cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -150,6 +153,7 @@ static void sc_populate_libgl_with_hostfs_symlinks(const char *libgl_dir,
 		char symlink_target[512] = { 0 };
 		char prefix_dir[512] = { 0 };
 		const char *pathname = glob_res.gl_pathv[i];
+		debug("pathname: %s", pathname);
```

Comment:
Can we please drop all these debug messages? Almost all of them won't make sense out of context.